docs/types/value-types.rst

```diff
@@ -157,23 +157,62 @@ Fixed Point Numbers
+The largest and the smallest number representable in a ``fixed16x2`` are ``327.67`` and ``-327.69``, respectively.
+
+For a fixed point type ``X``, you can use ``type(X).min`` and ``type(X).max`` to
+access the minimum and maximum value representable by the type, e.g. ``type(fixed16x2).max`` is ``327.67``.
+
+
+
+Type conversions:
+
+Implicit conversions from integer and fixed point types to ``fixedMxN`` types can be performed,
+as long as all values can still be represented ("widening" conversion).
+
+Implicit conversion from rational number expressions (literals) to fixed point types are only possible if there
+is no precision loss and the number is within the value range. This means that 0.01 can be implicitly
+converted to ``fixed64x2`` but not to ``fixed64x1``.
+
+Explicit conversions from rational number expressions (literals) are always possible as long as the
+value is within the range of the fixed point type.
+
+.. note::
+    Expressions like ``1/3``, which do not fit finite precision, always have to be explicitly converted to
+    a fixed point type to specify the desired precision. Use ``fixed64x4(1/3)`` for example to have
+    four decimal digits of precision.
+
+Explicit conversions between fixed point types can be used to reduce precision (resulting in rounding
+towards zero) or value range (resulting in removing higher order bits), but only
+one of value range, signedness and precision can be changed at a time.
+
+Explicit conversions to integer types truncate the fractional part (towards zero) and can only be performed if the integer
+type can contain the resulting value after truncation.
+
+Explicit conversions to and from ``bytesNN`` types of matching width can be performed and do not change the internal representation.
+
+
+
 Operators:
 
 * Comparisons: ``<=``, ``<``, ``==``, ``!=``, ``>=``, ``>`` (evaluate to ``bool``)
 * Arithmetic operators: ``+``, ``-``, unary ``-``, ``*``, ``/``, ``%`` (modulo)
 
 .. note::
-    The main difference between floating point (``float`` and ``double`` in many languages, more precisely IEEE 754 numbers) and fixed point numbers is
-    that the number of bits used for the integer and the fractional part (the part after the decimal dot) is flexible in the former, while it is strictly
-    defined in the latter. Generally, in floating point almost the entire space is used to represent the number, while only a small number of bits define
-    where the decimal point is.
+    Arithmetic operators are not yet supported.
+
+.. note::
+    Many programming language use floating point numbers (``float`` and ``double``, more precisely IEEE 754 numbers)
+    to handle fractional values. Floating point numbers use a different mechanism that makes it possible to represent
+    numbers close to zero with higher precision. Since the exact precision is harder to predict, though, they are
```

Comment:
> Floating point numbers use a different mechanism that makes it possible to represent
    numbers close to zero with higher precision

I don't really agree with this.

Reply:
Yes, it depends on whether you mean absolute or relative precision. I'll try to find a better wording.